modules/user_accounts/php/NDB_Form_user_accounts.class.inc

```diff
@@ -210,7 +210,8 @@ class NDB_Form_user_accounts extends NDB_Form
             // user name
             $group[] = $this->createText('UserID', 'User name');
             // $group[] = $this->form->createElement('static', null, null, '<label>');
-            $group = array_merge($group, $this->createCheckbox('NA_UserID', "Make user name match email address"));
+            //$group = array_merge($group, $this->createCheckbox('NA_UserID', "Make user name match email address"));
```

Comment:
Right. Cleaner that way.

On Tue, Apr 21, 2015 at 10:29 AM, mohadesz <notifications@github.com> wrote:

> In modules/user_accounts/php/NDB_Form_user_accounts.class.inc
> <https://github.com/aces/Loris/pull/1052#discussion_r28782863>:
>
> > @@ -210,7 +210,8 @@ class NDB_Form_user_accounts extends NDB_Form
> >              // user name
> >              $group[] = $this->createText('UserID', 'User name');
> >              // $group[] = $this->form->createElement('static', null, null, '<label>');
> > -            $group = array_merge($group, $this->createCheckbox('NA_UserID', "Make user name match email address"));
> > +            //$group = array_merge($group, $this->createCheckbox('NA_UserID', "Make user name match email address"));
>
> shouldn't be removed?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aces/Loris/pull/1052/files#r28782863>.
>